Sentinel/Classes/Core/PreferencesInfo/Views/PreferencesView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PreferencesView.swift
+//  Pods
+//
+//  Created by Zvonimir Medak on 17.03.2025..
+//
+
+import SwiftUI
+
+struct PreferencesView: View {
+
+    let preferenceItem: any PreferenceItem
```

Comment:
To make the naming more consistent throughout the library.
```suggestion
    let item: any PreferenceItem
```